chatops/test-hubot-st2-badconfig.bats

```diff
@@ -0,0 +1,105 @@
+
+load '../test_helpers/bats-support/load'
+load '../test_helpers/bats-assert/load'
+
+
+ST2_CHATOPS_ENV_FILE='/opt/stackstorm/chatops/st2chatops.env'
+
+@test "SETUP: setup environment variables for authentication related variables." {
+	`sed -e '/ ST2_API_KEY=/s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_TOKEN= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_USERNAME= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
```

Comment:
Can you please help me to understand what's the logic/reason behind this block and why do we need it? I found test description not clear enough

Reply:
@armab During `st2chatops` starts up, `st2chatops.env` is source’d directly and overwrites environment variable value passed from command line unless the it is not enabled.
In order to test different authentication configurations, disable all authentication related variables before test and revert changes after test.

Reply:
OK, after looking closer at tests you want to `unset` those ENV variables in `st2chatops.env` to verify edge cases.

Please make `@test` comment more clear as `setup environment variables for authentication related variables` is too blurry.